mesonbuild/linkers.py

```diff
@@ -230,6 +230,18 @@ def get_output_args(self, target: str) -> T.List[str]:
     def get_linker_always_args(self) -> T.List[str]:
         return ['rcs']
 
+class CompCertLinker(StaticLinker):
+
+    def __init__(self, exelist: T.List[str]):
+        super().__init__(exelist)
+        self.id = 'ccomp'
+
+    def can_linker_accept_rsp(self) -> bool:
+        return False
+
+    def get_output_args(self, target: str) -> T.List[str]:
+        return ['-o {}'.format(target)]
```

Comment:
This seems off to me, this would be equivalent to in bash doing:
```sh
$CC "-o ${target}"
```
I'm assuming you either want to pas this as a two element list, or to remove the space between them?